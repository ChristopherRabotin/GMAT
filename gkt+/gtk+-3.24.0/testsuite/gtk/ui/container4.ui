<!-- missing focus chain widget name attribute -->
<interface>
  <object class="GtkBox">
    <focus-chain>
      <widget/>
    </focus-chain>
  </object>
</interface>
