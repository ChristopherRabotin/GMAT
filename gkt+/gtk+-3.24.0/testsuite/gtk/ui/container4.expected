ERROR: g-markup-error-quark 6
.:5:16 element 'widget' requires attribute 'name'
